core/src/css/themes/high-contrast.scss

```
@@ -0,0 +1,160 @@
+@use "sass:map";
+@import "../../themes/ionic.functions.color";
+
+$primary:         #003FAE !default;
+$secondary:       #01487B !default;
+$tertiary:        #3400e6 !default;
+$success:         #004314 !default;
+$warning:         #5F4100 !default;
+$danger:          #9C000C !default;
+$light:           #f4f5f8 !default;
+$medium:          #444446 !default;
+$dark:            #222428 !default;
```

```suggestion
$primary:         #003fae !default;
$secondary:       #01487b !default;
$tertiary:        #3400e6 !default;
$success:         #004314 !default;
$warning:         #5f4100 !default;
$danger:          #9c000c !default;
$light:           #f4f5f8 !default;
$medium:          #444446 !default;
$dark:            #222428 !default;
```

Optional nit pick but we should keep these all the same case